samples/interactive-tutorials/src/main/java/events/RejoinUserEvent.java

```diff
@@ -40,11 +40,11 @@ public class RejoinUserEvent {
 
   public static void main(String[] args)
       throws IOException, ExecutionException, InterruptedException {
-    // TODO(developer): Replace these variables before running the sample.
-    String projectId = ServiceOptions.getDefaultProjectId();
+    // TODO: Set projectId to your Google Cloud Platform project ID.
+    String projectId = "my-project";
     String defaultCatalog =
         String.format("projects/%s/locations/global/catalogs/default_catalog", projectId);
-    // visitorId generated randomly.
+
     String visitorId = UUID.randomUUID().toString();
 
     callRejoinUserEvents(defaultCatalog, visitorId);
```

Comment:
Do we want to rename it to `rejoinUserEvents`

Reply:
In my opinion callRejoinUserEvents is descriptive of what the method is doing: calling the rejoinUserEvents method. So I don't think it is necessary for the name to change.